src/box/lua/info.c

```diff
@@ -650,6 +663,22 @@ lbox_schema_version(struct lua_State *L)
 	return 1;
 }
 
+/* gethostname() Lua interface inside box.info. */
+static int
+lbox_info_hostname(struct lua_State *L)
+{
+	char buffer[TT_HOST_NAME_MAX + 1];
+	int status = gethostname(buffer, TT_HOST_NAME_MAX + 1);
```

Comment:
```suggestion
	int status = gethostname(buffer, sizeof(buffer));
```

nit: probably it looks a bit better.

Reply:
Good idea, thanks.